### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by JordanMartinez**

### CODE REVIEW

**Comment by TomasMikula** [comment]
Are you able to change the signature of `anyOf` to

```java
<T extends Event, U extends T> EventPattern<T, U> anyOf(EventPattern<T, ? extends U>... events)
```

?

**Comment by JordanMartinez** [comment]
Yeah, I can change it to that. I just thought that `<Event, T extends Event>` would be able to incorporate more EventPatterns than the way you've suggested. For example,
````java
consume(
    anyOf(
        keyPressed(),
        mouseClicked()
    ), <lambda expression>);
````
If I change it to the way you're suggesting, then wouldn't `anyOf` only work for those events which extend the same base (e.g. KeyEvent)?

**Comment by JordanMartinez** [comment]
Hmm... I tried force-pushing your suggested change to keep the change in one commit. not sure why it didn't override the previous one.

**Comment by TomasMikula** [comment]
`EventPattern<T, U>` is able to test events of type `T`, e.g. `InputEvent`, and if they match, return a subtype of `T`, e.g. `KeyEvent`. If `anyOf` took `EventPattern<Event, U>`, you would not be able to pass `EventPattern<InputEvent, KeyEvent>` to it.

**Comment by TomasMikula** [comment]
Oh, and with

```java
<T extends Event, U extends T> EventPattern<T, U> anyOf(EventPattern<T, ? extends U>... events)
```

`T` can always be instantiated to `Event`, if that is desired.

File: src/main/java/org/fxmisc/wellbehaved/event/experimental/EventPattern.java [hunk 1]
```diff
@@ -59,24 +59,27 @@
         return onlyIf(condition.negate());
     }
 
-    static <T extends Event, U extends T, V extends U, W extends U> EventPattern<T, U> anyOf(
-            EventPattern<T, V> p1, EventPattern<T, W> p2) {
+    @SafeVarargs
+    static <T extends Event, U extends T> EventPattern<T, U> anyOf(EventPattern<T, U>... events) {
```
Comment (TomasMikula) [inline src/main/java/org/fxmisc/wellbehaved/event/experimental/EventPattern.java]: Can it be changed to

```
anyOf(EventPattern<T, ? extends U>... );
```

i.e. add `? extends` to the `U` parameter (only for the argument, not the return type).

**Comment by JordanMartinez** [comment]
How's that?

**Comment by TomasMikula** [comment]
👍

**Pull request merged**